README.md

```diff
@@ -63,6 +63,7 @@ const config = {
 | `region`     | Sets the region. <br> Default: `us-west-1`                                                       | `us-west-1` \| `eu-central-1`  |
 | `upload`     | Whether to upload report and assets to Sauce Labs. <br> Default: `true`                          | `boolean`                      |
 | `outputFile` | The local path to write the Sauce test report. Can be set in env var `SAUCE_REPORT_OUTPUT_NAME`. | `string`                       |
+| `mergeVideos` | Whether to merge all video files generated by Playwright. This is useful when used with the `upload` option to upload the test report to Sauce Labs since it will allow you to view the merged video in the Sauce Labs Test Results page. **Requires ffmpeg to be installed.**<br> Default: `false` | `boolean`
```

Comment:
You ran out of pipe there 😄 
```suggestion
| `mergeVideos` | Whether to merge all video files generated by Playwright. This is useful when used with the `upload` option to upload the test report to Sauce Labs since it will allow you to view the merged video in the Sauce Labs Test Results page. **Requires ffmpeg to be installed.**<br> Default: `false` | `boolean` |
```